bluepyefe/extract.py

```diff
@@ -214,7 +215,7 @@ def extract_efeatures_at_targets(
     return list(cells)
 
 
-def compute_rheobase(cells, protocols_rheobase, spike_threshold=1):
+def compute_rheobase(cells, protocols_rheobase, rheobase_strategy="absolute"):
```

Comment:
```suggestion
def compute_rheobase(cells, protocols_rheobase, rheobase_strategy="absolute", **rheo_kwargs):
```

We should let the user pass args for the rheobase function